hist/histv7/inc/ROOT/RHist.hxx

```diff
@@ -333,6 +340,81 @@ void Add(RHist<DIMENSIONS, PRECISION, STAT_TO...> &to, const RHist<DIMENSIONS, P
    toImpl.GetStat().Add(fromImpl.GetStat());
 }
 
+/// Divide the `to` histogram by `from`.
+///
+/// `to = to/from`
+/// This operation may currently only be performed if the two histograms have
+/// the same axis configuration, use the same precision, and if `from` records
+/// at least the same statistics as `to` (recording more stats is fine).
+///
+/// Dividing histograms with incompatible axis binning will be reported at runtime
+/// with an `std::runtime_error`. Insufficient statistics in the source
+/// histogram will be detected at compile-time and result in a compiler error.
+///
+/// In the future, we may either adopt a more relaxed definition of histogram
+/// division or provide a mechanism to convert from one histogram type to
+/// another. We currently favor the latter path.
```

Comment:
Interesting to put this into the doc - for an experimental interface that makes a lot of sense!